### PR TITLE
New brain damage line

### PR DESCRIPTION
**Pull request by BeeSting12**
Why: It's funny.

"THE FUCKING CHIEF ENGEENIR STUNNED ME AND PUT ME OUT OF ENGEENERING"

### CODE REVIEW

**Comment by tortellinitony** [comment]
why are these kind of PRs allowed

File: strings/traumas.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "brain_damage": [
         "@pick(semicolon)IM A PONY NEEEEEEIIIIIIIIIGH",
         "without oxigen blob don't evoluate?",
+		"THE FUCKING CHIEF ENGEENIR STUNNED ME AND PUT ME OUT OF ENGEENERING",
```
Comment (ninjanomnom) [inline strings/traumas.json]: Wrong indentation

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: strings/traumas.json (hunk 1 above)
Comment (BeeSting12) [reply strings/traumas.json]: It doesn't show up that way for me

Comment (ninjanomnom) [reply strings/traumas.json]: You're using tabs instead of spaces for indentation is the issue. This file is space indented

**Comment by BeeSting12** [comment]
Can I get a five paragraph explanation on why this was closed

**Comment by optimumtact** [comment]
no

**Comment by BeeSting12** [comment]
Understandable, have a good day.

**Pull request closed without merging**